src/Dictionaries/CacheDictionaryUpdateQueue.h

```diff
@@ -147,23 +152,29 @@ class CacheDictionaryUpdateQueue
     */
     void waitForCurrentUpdateFinish(CacheDictionaryUpdateUnitPtr<dictionary_key_type> & update_unit_ptr) const;
 
+    /// Get current update queue size
+    size_t getSize() const;
```

Comment:
Looks like there is no need in exporting this for public use?

Reply:
Fixed.